### PR TITLE
fix: playwright fix

### PR DESCRIPTION
**Pull request by keita-determined**
## Description
fix react e2e test failure https://app.circleci.com/pipelines/github/determined-ai/determined/49978/workflows/c3146d0f-01a7-4efa-8c3e-4206f537ca74/jobs/2191778

```
Error: browserType.launch: Executable doesn't exist at /ms-playwright/webkit-1944/pw_run.sh
    ╔═════════════════════════════════════════════════════════════════════════╗
    ║ Looks like Playwright Test or Playwright was just installed or updated. ║
    ║ Please run the following command to download new browsers:              ║
    ║                                                                         ║
    ║     npx playwright install                                              ║
    ║                                                                         ║
    ║ <3 Playwright Team                                                      ║
    ╚═════════════════════════════════════════════════════════════════════════╝
```

## Test Plan
- Check if react e2e test passes

## Checklist

- [x] Changes have been manually QA'd
- [ ] User-facing API changes need the "User-facing API Change" label.
- [ ] Release notes should be added as a separate file under `docs/release-notes/`.
  See [Release Note](https://github.com/determined-ai/determined/blob/master/docs/release-notes/README.md) for details.
- [ ] Licenses should be included for new code which was copied and/or modified from any external code.

### CODE REVIEW

**Comment by keita-determined** [comment]
Unrelated ci, `ci/circleci: test-unit-storage`, is blocking

File: .circleci/real_config.yml [hunk 1]
```diff
@@ -1530,7 +1530,12 @@ jobs:
       - skip-if-github-only
       - run: apt-get update && apt-get -y install make
       - react-get-deps
-      - run: npx playwright install --with-deps && npx playwright install chrome
+      - run:
+          name: Install Playwright
+          command: |
+            cd ./webui/react
+            npx playwright install --with-deps && npx playwright install chrome
+            cd -
```
Comment (hkang1) [inline .circleci/real_config.yml:1538]: **suggestion:** wdy think about making this a target in `webui/react/Makefile` such that you can just call `make -C web/react install-playwright` here?

Comment (keita-determined) [reply .circleci/real_config.yml:1538]: i was thinking about doing it, but the command is only used in ci (for local install, just `npx playwright install`), but i'll change it with make command

**Review by dannysauer — APPROVED** [review]
Weird that this fixes it. Seems to work though, so great! :D

**Review by hkang1 — APPROVED** [review]
LGTM! thanks!

**Pull request merged**